PatternPal/PatternPal.LoggingServer/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+using Microsoft.EntityFrameworkCore;
+using PatternPal.LoggingServer.Data;
+using PatternPal.LoggingServer.Data.Interfaces;
+using PatternPal.LoggingServer.Models;
+using PatternPal.LoggingServer.Services;
+
+namespace PatternPal;
+
+internal static class Program
+{
+    internal static void Main()
+    {
+        WebApplicationBuilder builder = WebApplication.CreateBuilder();
+
+        builder.Services.AddGrpc();
+
+
+
```

Comment:
This is more of a nit, and I wouldn't block merging over this, but these empty lines are unnecessary. There are more occurrences of this throughout the PR.